### PR TITLE
Add tests for adding licence during creating of job or edit

### PR DESCRIPTION
**Pull request by sigu**
Closes AgileVentures/MetPlus_tracker#649

**Done:**
- fixing of the file styling
- add tests for adding licences when job is added 
- add tests for adding licences when job is edited
- tests for removing one of the licenses

### CODE REVIEW

File: features/jobs.feature [hunk 1]
```diff
@@ -1,216 +1,244 @@
...
+      | Description    | At least two years work experience |
+    And I select "Day" in select list "Shift"
+    And  I select "16 Fall Detroit, Michigan 02074" in select list "Job Location"
+    And I select "Full Time" in select list "Job Type"
+    And I select "Part Time" in select list "Job Type"
+    And I select "Morning" in select list "Shift"
+    Then I select a license
+    And I press "new-job-submit"
+    Then I should see "cashier has been created successfully."
+    And the job "cashier" should have 1 license
```
Comment (patmbolger) [inline features/jobs.feature]: Please add this line here:
```
And I should see "LLMSW (LIMITED LICENSE MASTER SOCIAL WORKER)"
```

File: features/jobs.feature [hunk 2]
```diff
@@ -1,216 +1,244 @@
...
+    And I fill in "Additional Information:" with "also need XYZ training"
+    And I press "Update"
+    Then I should see "cab-driver has been updated successfully."
+    And I should see "Associates Degree"
+    And I should see "also need XYZ training"
+
+    Then I click the "Edit Job" link
+    And I select another license
+    And I press "Update"
+    And the job "cab-driver" should have 2 licenses
```
Comment (patmbolger) [inline features/jobs.feature]: Please add these lines:
```
And I should see "LLMSW (LIMITED LICENSE MASTER SOCIAL WORKER)"
And I should see "LLPC (LIMITED LICENSE PROFESSIONAL COUNSELOR)"
```

File: features/jobs.feature [hunk 3]
```diff
@@ -1,216 +1,244 @@
...
+
+    Then I click the "Edit Job" link
+    Then I wait for 1 second
+    And I click the first "remove license" link
+    And I press "Update"
+    And the job "cab-driver" should have 1 license
```
Comment (patmbolger) [inline features/jobs.feature]: Please add this line:
```
And I should see "LLPC (LIMITED LICENSE PROFESSIONAL COUNSELOR)"
```

**Comment by sigu** [comment]
Corrected @patmbolger , thanks for the comprehensive feedback

**Pull request merged**